src/applications/personalization/profile/components/personal-health-care-contacts/NonVAPatientMessage.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+import PropTypes from 'prop-types';
+
+const NonVAPatientMessage = ({ testId }) => (
+  <>
+    <p data-testid={testId}>
+      Our records show that you don’t currently receive health care benefits
+      from the VA.
+    </p>
+    <p>
+      If you think this is an error, call us at{' '}
+      <va-telephone contact={CONTACTS.VA_BENEFITS} />
+      <va-telephone contact={CONTACTS['711']} tty />. We’re here Monday through
```

Comment:
Put some parentheses around the 711 `<va-telephone />` component so that we match the copy in the designs.

Reply:
Another option would be to use the `<HelpDeskContact/>` component.

Reply:
thank you, made this change

Reply:
fixed the phone number display